buildSrc/src/main/java/se/nbis/lega/deployment/LocalEGATask.java

```diff
@@ -176,7 +198,11 @@ protected void createConfig(String name, File file) throws IOException {
     }
 
     protected String getMachineIPAddress(String name) throws IOException {
+        // if (machineIp == null) {
```

Comment:
If these lines should not be here anymore, just remove them, no need to comment out.